### PR TITLE
shortened delay to notifications upon tracking course

### PR DESCRIPTION
**Pull request by emarc0314**
## Overview

Delayed the time duration of a notification.

## Changes Made

Added a DispatchQueue.main.asyncAfter(){} that lasts 1.15 seconds in SearchDetailTableViewController.

## Test Coverage

Tracked various courses and untracked them to make sure the duration was accurate.

## Screenshots (delete if not applicable)

<details>

  <summary>Notifications</summary>

IMG_2BD28A0FD6C5-1  

</details>

### CODE REVIEW

File: CourseGrab/ViewControllers/SearchDetail/SearchDetailTableViewController.swift [hunk 1]
```diff
@@ -89,7 +89,11 @@ extension SearchDetailTableViewController {
                         self.tableView.reloadRows(at: [indexPath], with: .automatic)
                         let banner = NotificationBanner(customView: TrackingBannerView(section: section, isTracking: true))
                         banner.show(queuePosition: .front, queue: bannerQueue)
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 1.15) {
+                            banner.dismiss()
+                        }
                     }
+
```
Comment (danielVebman) [inline CourseGrab/ViewControllers/SearchDetail/SearchDetailTableViewController.swift:96]: delete newline

**Review by spd75 — APPROVED** [review]
Absolute gas

**Review by haiyingweng — APPROVED** [review]
🔥

File: CourseGrab/ViewControllers/SearchDetail/SearchDetailTableViewController.swift [hunk 2]
```diff
@@ -105,7 +109,11 @@ extension SearchDetailTableViewController {
                         self.tableView.reloadRows(at: [indexPath], with: .automatic)
                         let banner = NotificationBanner(customView: TrackingBannerView(section: section, isTracking: false))
                         banner.show(queuePosition: .front, queue: bannerQueue)
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 1.15) {
+                            banner.dismiss()
+                        }
                     }
+
```
Comment (danielVebman) [inline CourseGrab/ViewControllers/SearchDetail/SearchDetailTableViewController.swift:116]: delete

**Review by danielVebman — APPROVED** [review]
lgtm!

**Review by Npikielny — APPROVED** [review]
Looks good to me!

**Pull request merged**